src/pages/AdminLogin.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {FaGoogle,FaRegEnvelope} from 'react-icons/fa'
+import {MdLockOutline} from 'react-icons/md'
+
+const AdminLogin = () => {
+  return (
+    <>
+      <div className="md:flex md:flex-col md:items-center md:justify-center w-full  min-h-screen  text-center py-2 bg-gray-100 sm:flex sm:flex-row sm:items-center sm:justify-center">
```

Comment:
Because the header is fixed, some content is being hidden. You can probably decrease the height of the card and add some margin at the top.

Reply:
Also, remember to change the background color on the dark mode.